# Data path. adjust them!
common:
    from_bag: true   # if true, use rosbag to generate pcd file, then use them to calibrate, else use pcd file directly
    image_path:  "/home/squirtle/Desktop/Squirtle/cv/ws_livox/data/basic1_and_advance4_calibration/image"
    bag_path:  "/home/squirtle/Desktop/Squirtle/cv/ws_livox/data/basic1_and_advance4_calibration/livox_bag"
    save_pcd_path:  "/home/squirtle/Desktop/Squirtle/cv/ws_livox/data/basic1_and_advance4_calibration/my_pcd"
    pcd_path:  "/home/squirtle/Desktop/Squirtle/cv/ws_livox/data/basic1_and_advance4_calibration/pcd"
    result_path:  "/home/squirtle/Desktop/Squirtle/cv/ws_livox/data/basic1_and_advance4_calibration/extrinsic.txt"
    data_num: 7
# Camera Parameters. Adjust them!
camera:
    camera_matrix:  [910.623, 0.0, 651.66,
                     0.0,  910.183, 365.568,
                     0.0, 0.0, 1.0 ] 
    dist_coeffs: [0.0, 0.0, 0.0, 0.0, 0.0]

# Calibration Parameters.!
calib:
    calib_config_file: "/home/squirtle/Desktop/Squirtle/cv/ws_livox/src/livox_camera_calib/config/config_indoor.yaml"
    use_rough_calib: true # set true if your initial_extrinsic is bad
